### DISCUSSION TITLE
pdf forms

### DISCUSSION THREAD

**Discussion by tonyappleton** [Stirling-Tools/Stirling-PDF#727] [Ideas]
Heard about this project on Jupiter Broadcasting's Self-Hosted podcast and had to give it a try! Brilliant - thank you for this - been trying to find ways to avoid the Adobe Tax for a long time. Any plans to add/manipulate form fields?

**Comment by sbplat** [comment]
I believe that the [add-elements](/Stirling-Tools/Stirling-PDF/tree/add-elements) branch implements this feature, but is still being worked on.
